hasura/metadata/tables.yaml

```diff
@@ -119,5 +119,5 @@
     permission:
       columns:
       - id
-      - eos_price
```

Comment:
what about the eos price are we still persisting that?

Reply:
@xavier506  `eos_price` has been renamed to `token_price`